examples/pytorch_lightning/pmsn.py

```diff
@@ -87,21 +85,16 @@ def configure_optimizers(self):
 
 model = PMSN()
 
-# we ignore object detection annotations by setting target_transform to return 0
-pascal_voc = torchvision.datasets.VOCDetection(
-    "datasets/pascal_voc", download=True, target_transform=lambda t: 0
-)
 transform = MSNTransform()
-dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=transform)
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
+)
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?